benches/mkbench.rs

```diff
@@ -153,6 +192,15 @@ impl<'a, T: Dispatch + Sync + Default> ReplicaTrait for Replica<'a, T> {
     ) -> <Self::D as Dispatch>::Response {
         self.execute(op, idx)
     }
+
+    #[cfg(feature = "nr")]
+    async fn async_exec_ro(
+        &self,
+        op: <Self::D as Dispatch>::ReadOperation,
+        idx: ReplicaToken,
+    ) -> Pin<Box<dyn futures::Future<Output = <Self::D as Dispatch>::Response> + 'life0>> {
+        Box::pin(self.async_execute(op, idx).await)
```

Comment:
maybe explore passing in Option<Box::pin> as discussed to potentially avoid allocation?